_entries/2016-05-04-hyperlinks.md

```diff
@@ -35,16 +35,18 @@ Not this
 
 Don’t use meaningless terms such as ‘click here’, ‘read more’ or ‘useful links’.
 
-### Drop the http://
+### Drop the http://www
 
-When referencing a website, drop the `http://` prefix.
+When referencing a website, drop the `http://www` prefix. Verify the website doesn't need `www` to reach it.
```

Comment:
Hey @klepas

Are you saying they should use the full URL even if there is an auto redirect in place, just to be safe?

I'm trying to make this as concise as possible. Happy to be more verbose if necessary. What do you think of this?

```
If you can reach the website without using `http://www` you don't have to reference it. But you need to use the full URL when you create a hyperlink.
```